app/controllers/application_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
+  include ApplicationHelper
+    helper_method :current_user
+    helper_method :logged_in?
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
```

Comment:
Adding `if session[:user_id]` at the end of the line would prevent unnecessary queries